### PR TITLE
Refine trace and logs SDK configuration sections

### PR DESCRIPTION
**Pull request by pellared**
Follows https://github.com/open-telemetry/opentelemetry-specification/pull/3522

Fixes https://github.com/open-telemetry/opentelemetry-specification/issues/3523

Fixes https://github.com/open-telemetry/opentelemetry-specification/issues/3532

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,13 +11,19 @@ release.
 
 ### Traces
 
+- Refine SDK TracerProvider configuration section.
+  ([#3522](https://github.com/open-telemetry/opentelemetry-specification/pull/3559))
```
Comment (marcalff) [inline CHANGELOG.md:15]: In the link, the PR number should be 3559, not 3522

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,13 +11,19 @@ release.
...
+
 ### Metrics
 
 - Refine SDK MeterProvider configuration section.
   ([#3522](https://github.com/open-telemetry/opentelemetry-specification/pull/3522))
 
 ### Logs
 
+- Refine SDK LoggerProvider configuration section.
+  ([#3522](https://github.com/open-telemetry/opentelemetry-specification/pull/3559))
```
Comment (marcalff) [inline CHANGELOG.md:25]: In the link, the PR number should be 3559, not 3522

**Review by marcalff — COMMENTED** [review]
Some copy and paste issues in the CHANGELOG

**Review by MrAlias — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pellared) [reply CHANGELOG.md:15]: Nice catch!

File: CHANGELOG.md (hunk 2 above)
Comment (pellared) [reply CHANGELOG.md:25]: Nice catch!

File: CHANGELOG.md (hunk 1 above)
Comment (pellared) [reply CHANGELOG.md]: https://github.com/open-telemetry/opentelemetry-specification/pull/3559/commits/4f6a39b7035a4bf3f762bb8f245eb7edf5cb96ca

File: CHANGELOG.md (hunk 2 above)
Comment (pellared) [reply CHANGELOG.md]: https://github.com/open-telemetry/opentelemetry-specification/pull/3559/commits/4f6a39b7035a4bf3f762bb8f245eb7edf5cb96ca

**Review by jack-berg — APPROVED** [review]

**Review by marcalff — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Pull request merged**